docs/user-contributed/py3status.html

```diff
@@ -8,143 +8,203 @@ <h1>User-contributed article: enhanced and extensible i3bar with py3status</h1>
 
 <p>
 In the i3 documentation, the recommended tool for <a
-href="http://i3wm.org/docs/userguide.html#_displaying_a_status_line">displaying
+href="https://i3wm.org/docs/userguide.html#_displaying_a_status_line">displaying
 a status line is to use i3status</a> combined with i3bar.
 </p>
 
 <p>
-While i3status is very efficient at what it does, it is by design limited to
-a few modules and does not allow you to inject your own scripts output on your
-i3bar. This is said pretty clearly on the i3status man page:
+While i3status is very efficient at what it does, it is, by design, fairly
+limited to a few modules and does not allow not allow the users to add anything
+to the i3bar. This was said pretty clearly on the i3status man page:
```

Comment:
This isn't actually quite true anymore: https://github.com/i3/i3status/pull/331#issuecomment-464344432. Since we're updating this, maybe we should update it to this new information, including removing the quote below? :-)

This of course doesn't take away from the usefulness of py3status, which is a great tool. Printing from file alone doesn't solve all use-cases py3status does.

Reply:
Thanks for pointing that out and for being kind to py3status mate.

I've rephrased to take your point into account.